ddl/ddl_api.go

```diff
@@ -58,11 +58,20 @@ import (
 
 const expressionIndexPrefix = "_V$"
 
+type charsetCollate struct {
```

Comment:
What's different between `charsetCollate ` and `ast.CharsetOpt` struct?

Reply:
Good catch! I did try to look for a struct like `ast.CharsetOpt` but failed to get one... so `charsetCollate` is not needed.